pkgs/development/androidndk-pkgs/androidndk-pkgs.nix

```
@@ -48,35 +48,52 @@ let
   hostInfo = ndkInfoFun stdenv.hostPlatform;
   targetInfo = ndkInfoFun stdenv.targetPlatform;
 
-  prefix = lib.optionalString (stdenv.targetPlatform != stdenv.hostPlatform) (stdenv.targetPlatform.config + "-");
+  sdkVer = stdenv.targetPlatform.sdkVer;
+  prefix = lib.optionalString (stdenv.targetPlatform != stdenv.hostPlatform) (stdenv.targetPlatform.config);
```

```suggestion
  # targetInfo.triple is what Google thinks, this is the slightly different one we use.
  # We make it 4 parts to conform to the existing standard more properly.
  targetConfig = stdenv.targetPlatform.config;
```
If this becomes `""` the stuff below will not work. Also without `-` I don't think we should call it `prefix`, but feel free to call it something other than `targetConfig`.

---

ok. I'll make this change.

---

done!